tests/fixtures/lsp/hover/command.nu

```diff
@@ -1,6 +1,7 @@
 # Renders some greeting message
-def hello [] {}
+def hello [] { }
 
 hello
 
 [""] | str join
+^sleep
```

Comment:
@fdncred not sure sleep exists by default on windows or not, if not you can add new commands below, then adjust the row number in the test case accordingly.

Reply:
Sleep does not exist on Windows but does within powershell. I'll see what I can do.

Reply:
I updated it to use powershell on Windows. Please feel free to tweak, change, remove whatever you think is dumb. :)